content/SCALE/SCALETutorials/Reporting/ConfigReportsScale.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Configuring Reporting"
+description: "This article provides information on changing settings that control how TrueNAS displays report graphs, interacting with graphs, and the TrueCommand Enhancement option."
+weight: 10
+alias: 
+tags:
+ - scalereports
+---
+
+{{< toc >}}
+
+TrueNAS has a built-in reporting engine that provides helpful graphs and information about the system.
+
+![ReportingCPUGraphs1SCALE](/images/SCALE/22.02/ReportingCPUGraphs1SCALE.png "CPU Reporting Usage and Temperature")
+
+{{< expand "What does TrueNAS use for reporting?" "v" >}}
+TrueNAS uses [Graphite](https://graphiteapp.org/ "Graphite Homepage") to gather metrics and create visualizations.
+
+TrueNAS uses [collectd](https://collectd.org/) to provide reporting statistics. 
+{{< /expand >}}
+
+Reporting data is saved to permit viewing and monitoring usage trends over time.
+This data is preserved across system upgrades and restarts.
+
+{{< hint warning >}}
+Because reporting data is written frequently do not store it on the boot pool or operating system device.
+
+TrueNAS clears the report history when you change the report CPU, graph age, or graph points options.
+{{< /hint >}}
+
+Data files are saved in <file>/var/db/collectd/rrd/</file>.
+
+## Configuring Report Settings
+Click the <i class="material-icons" aria-hidden="true" title="Settings">settings</i> to open the **Reports Configuration** configuration screen where you control how TrueNAS displays the graphs.
+
+![ReportsConfigurationSCALE](/images/SCALE/22.02/ReportsConfigurationSCALE.png "Reports Configuration")
+
+Select the general options you want to used in your TrueNAS.
+
+Specify either the host name or IP address of the [Graphite](https://graphiteapp.org/) server you want to use.
```

Comment:
```suggestion
Specify either the hostname or IP address of the [Graphite](https://graphiteapp.org/) server you want to use.
```

Reply:
host name is two words unless you are talking about the CLI command hostname.